packages/vue/src/components/organisms/SfCollectedProduct/SfCollectedProduct.vue

```diff
@@ -3,9 +3,16 @@
     <slot name="remove" v-bind="{ removeHandler }">
       <SfCircleIcon
         icon="cross"
-        class="sf-collected-product__remove desktop-only"
+        class="sf-circle-icon--small sf-collected-product__remove sf-collected-product__remove--circle-icon"
```

Comment:
@martaradziszewska do we need two custom class `sf-collected-product__remove sf-collected-product__remove--circle-icon`?